src/components/AppStyle.tsx

```diff
@@ -29,6 +29,7 @@ const palette = {
   link: "#0094FF",
   linkDark: "#007FD6",
   warning: "#faad14",
+  successHeavy: "#6BD352",
```

Comment:
Chosen arbitrarily as a hover color between the default success color and the `successMedium`.